src/main/java/net/fabricmc/fabric/block/FabricBlockSettings.java

```diff
@@ -141,6 +150,19 @@ public FabricBlockSettings strength(float hardness, float resistance) {
 		return this;
 	}
 
+	public FabricBlockSettings strength(float both) {
+		return strength(both, both);
+	}
+
+	public FabricBlockSettings breakInstantly() {
+		return strength(0.0F);
+	}
+
+	public FabricBlockSettings isPistonExtension() {
+		this.pistonExtension = true;
```

Comment:
I don't think this field is defined anywhere, which means this doesn't compile.

Reply:
Yeah, I'll be handling this myself - I just want to know where pistonExtension came from as I am of high belief that's not Mojang's intention...

...right?

Reply:
Please do! I'm stalling this PR until then.

Reply:
The field is declared as a property in Block and Block.Settings, and initialized by Block.Settings. I have no idea what the implications are from it making BlockState.initShapeCache do nothing, and at a glance the additional logic it activates in BaseFluid doesn't actually do anything.

..and that's the extent of my knowledge.

Reply:
I remapped it to `hasDynamicBounds`/`dynamicBounds` in a recent PR, so think issue should be sorted now :)